Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -91,3 +91,4 @@
 --rules elseOnSameLine
 --rules sortTypealiases
 --rules preferForLoop
+--rules wrapMultilineConditionalAssignment
```

Comment:
After https://github.com/nicklockwood/SwiftFormat/pull/1574 is merged, I'll pull in an updated version of SwiftFormat that actually includes this rule implementation.